README.md

```diff
@@ -2,475 +2,299 @@
+*Burstable CPU performance* |  ✓  |  ✓  |      |  ✓   |  ✓  |
+*Isolated emulator threads* |     |     |  ✓   |      |     |  ✓
+*vNUMA*                     |     |     |  ✓   |      |     |
+*vCPU-To-Memory Ratio*      | 1:4 | 1:4 |  1:2 |  1:4 | 1:8 | 1:2
+
+## U Series
+
+The U Series is quite neutral and provides resources for
+general purpose applications.
+
+*U* is the abbreviation for "Universal", hinting at the universal
+attitude towards workloads.
+
+VMs of instance types will share physical CPU cores on a
+time-slice basis with other VMs.
+
+### U Series Characteristics
+
+Specific characteristics of this series are:
+- *Burstable CPU performance* - The workload has a baseline compute
+  performance but is permitted to burst beyond this baseline, if
+  excess compute is available
+- *vCPU-To-Memory Ratio (1:4)* - A vCPU-to-Memory ratio of 1:4, for less
+  noise per node
+
+## O Series
+
+The O Series is based on the U Series, with the only difference
+being that memory is overcommitted.
+
+*O* is the abbreviation for "Overcommitted".
+
+### UO Series Characteristics
+
+Specific characteristics of this series are:
+- *Burstable CPU performance* - The workload has a baseline compute
+  performance but is permitted to burst beyond this baseline, if
+  excess compute is available
+- *Overcommitted Memory* - Memory is over-committed in order to achieve
+  a higher workload density
+- *vCPU-To-Memory Ratio (1:4)* - A vCPU-to-Memory ratio of 1:4, for less
+  noise per node
+
+## CX Series
+
+The CX Series provides exclusive compute resources for compute
+intensive applications.
+
+*CX* is the abbreviation of "Compute Exclusive".
+
+The exclusive resources are given to the compute threads of the
+VM. In order to ensure this, some additional cores (depending
+on the number of disks and NICs) will be requested to offload
+the IO threading from cores dedicated to the workload.
+In addition, in this series, the NUMA topology of the used
+cores is provided to the VM.
+
+### CX Series Characteristics
+
+Specific characteristics of this series are:
+- *Hugepages* - Hugepages are used in order to improve memory
+  performance
+- *Dedicated CPU* - Physical cores are exclusively assigned to every
+  vCPU in order to provide fixed and high compute guarantees to the
+  workload
+- *Isolated emulator threads* - Hypervisor emulator threads are isolated
+  from the vCPUs in order to reduce emaulation related impact on the
+  workload
+- *vNUMA* - Physical NUMA topology is reflected in the guest in order to
+  optimize guest sided cache utilization
+- *vCPU-To-Memory Ratio (1:2)* - A vCPU-to-Memory ratio of 1:2
+
+## GN Series
+
+The GN Series provides instances types intended for VMs with
+NVIDIA GPU resources attached.
+
+*GN* is the abbreviation of "GPU NVIDIA".
+
+This series is intended to be used with VMs consuming GPUs
+provided by the
+[NVIDIA GPU Operator](https://github.com/NVIDIA/gpu-operator)
+which is made available on OpenShift via OperatorHub.
```

Comment:
Is this operator exclusive of OCP? If not, I would write something like: `which is also available on..`  Something that makes it clear that it can be used in both OCP and K8s

Reply:
Right, this operator also works on K8s. Updating the description.